nitter/nitter.php

```diff
@@ -38,6 +38,9 @@ function nitter_install()
 function nitter_addon_admin_post(App $a)
 {
 	$nitterserver = trim($_POST['nitterserver']);
```

Comment:
You could use `$nitterserver = rtrim(trim($_POST['nitterserver']), '/');` instead if the following two lines.

Reply:
done